v2.2/follower-reads.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Follower Reads
+summary: To reduce latency for read queries, you can choose to have the closest node serve the request using the "Follower reads" feature.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To reduce latency for read queries, you can choose to have the closest node serve the request using the follower read  feature. For this to work, though, the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.
```

Comment:
> For this to work, though, the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.

I would remove this sentence.  It's a bit confusing, and also the information is better expressed below (IMO).  I think having the intro be the single sentence "To reduce latency, ..." is fine.

Reply:
> have the closest node serve the request using the follower read  feature

s/read/reads/

I'm not sure if the singular is also correct, but it caught my eye since the page title is "Follower reads".